### ISSUE TITLE
ENUMs in arguments

### ISSUE THREAD

**Issue by Sexton-D** [mghoneimy/php-graphql-oqm#5]
I'm looking into using this in one of my projects, but I'm seeing that it doesn't properly support using Enums in query arguments.

for example, I have this schema snipit
```
queryName(OrderBy: [OrderByClause!]){
 ...
}

Input OrderByClause{
 field: String!
 order: SortOrder!
}

enum SortOrder{ 
 ASC
 DESC
}
```

But when that gets run through your generator, the OrderByClause's order field gets converted into an InputObject argurment, when its an enum. 

I am using Laravel and Lighthouse, if it helps any

**Comment by mghoneimy** [comment]
Hello @Sexton-D ,
From your schema I can clearly see that OrderByClause is an input object.
In the declaration it says `Input` right before the name declaration, it's not an Enum type.
The SortOrder is the one which is an `Enum` type and it's there in the declaration.
Let me know if you have any other questions

**Comment by Sexton-D** [comment]
Sorry, that was poorly explained. 

When it generates the php classes, the `->setOrder()` argument expects a `SortOrderInputObject`, but (obviously) none was created.
The `SortOrderEnumObject` was created correctly, however.

But digging into the code, there is no support for Enums as arguments for queries.

**Comment by mghoneimy** [comment]
@Sexton-D Now I understand your problem. I'll look into this tonight and get back to you.

**Comment by mghoneimy** [comment]
@Sexton-D Issue fixed in the v1.0.2 https://github.com/mghoneimy/php-graphql-oqm/commit/263915c1f0f917a29e1e164799712c94de3bf97b
Thank you for reporting this!
